src/DataProvider/OperationDataProviderTrait.php

```diff
@@ -99,6 +99,10 @@ private function extractIdentifiers(array $parameters, array $attributes)
             return $id;
         }
 
+        if (!isset($attributes['subresource_context'])) {
+            throw new RuntimeException("We could not recognize which operation you're using, either you don't want the data to be read (use _api_receive=false) or make sure you declared the operation name (\"item_operation_name\", \"collection_operation_name\").");
```

Comment:
Either "item_operation_name" or "collection_operation_name" must be defined, unless "_api_receive" is set to false.